web/src/AutoBoundariesMode.svelte

```diff
@@ -79,6 +88,46 @@
     }
     return x;
   }
+
+  function fillColor(
```

Comment:
I thought about sharing more code with pick neighbourhood, but:

1) We have different coloring for "none" between the two modes
2) The popup behavior is different

Duplicating doesn't seem too bad